pusher/src/Controller/AuthenticateController.ts

```diff
@@ -24,6 +21,26 @@ export class AuthenticateController extends BaseHttpController {
         this.me();
     }
 
+    roomAccess() {
+        //eslint-disable-next-line @typescript-eslint/no-misused-promises
+        this.app.get("/room/access", async (req, res) => {
+            try {
+                const { uuid, playUri } = parse(req.path_query);
+                if (!uuid || !playUri) {
+                    throw new Error("Missing uuid and playUri parameters.");
+                }
+                return res.json(
+                    await adminService.fetchMemberDataByUuid(uuid as string, playUri as string, req.ip, [])
+                );
+            } catch (e) {
+                console.warn(e);
+            }
+            res.status(500);
+            res.send("User cannot be identified.");
+            return;
+        });
```

Comment:
I think you can remove the warning by propertly handling an error case in the promise (running it into an anonymous function. Something like:

```suggestion
        this.app.get("/room/access", (req, res) => {
            (() => {
                try {
                    const { uuid, playUri } = parse(req.path_query);
                    if (!uuid || !playUri) {
                        throw new Error("Missing uuid and playUri parameters.");
                    }
                    return res.json(
                        await adminService.fetchMemberDataByUuid(uuid as string, playUri as string, req.ip, [])
                    );
                    } catch (e) {
                    console.warn(e);
                }
                res.status(500);
                res.send("User cannot be identified.");
                return;
            })().catch(e => console.error(e));
        });
```